imblearn/dask/_support.py

```diff
@@ -0,0 +1,13 @@
+_REGISTERED_DASK_CONTAINER = []
+
+try:
+    from dask import array, dataframe
```

Comment:
People can have just `dask[array]` installed (not dataframe) so it's possible to have the `array` import succeed, but the `dataframe` import fail. So if you want to support that case those would need to be in separate try / except blocks.

Maybe you instead want `from dask import is_dask_collection`? That's a bit broader though (it also covers anything implementing dask's collection interface like `dask.Bag`, `xarray.DataArray` and `xarray.Dataset`).

Reply:
That seems what I wanted :)